content/src/main/content/jcr_root/apps/acs-commons/components/utilities/report-builder/configs/paths-list/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+	jcr:primaryType="nt:unstructured"
+	jcr:title="Report Builder Config"
+	sling:resourceType="cq/gui/components/authoring/dialog"
+	helpPath="https://adobe-consulting-services.github.io/acs-aem-commons/features/reports">
+	<content
+		jcr:primaryType="nt:unstructured"
+		sling:resourceType="granite/ui/components/foundation/container">
+		<layout
+			jcr:primaryType="nt:unstructured"
+			sling:resourceType="granite/ui/components/foundation/layouts/fixedcolumns"/>
+		<items jcr:primaryType="nt:unstructured">
+			<column
+				jcr:primaryType="nt:unstructured"
+				sling:resourceType="granite/ui/components/foundation/container">
+				<items jcr:primaryType="nt:unstructured">
```

Comment:
It might be worth adding fieldDescriptions to these as well; TBH im not entirely clear what "Page Size" does.. and i only know that the paths are linebreak delimited because of the screenshot.

Reply:
fixed